packages/spor-input-react/src/Input.tsx

```diff
@@ -34,19 +34,22 @@ export type InputProps = Exclude<ChakraInputProps, "variant" | "size"> & {
  * ```
  */
 export const Input = forwardRef<InputProps, "input">(
-  ({ label, leftIcon, rightIcon, ...props }, ref) => {
+  ({ label, leftIcon, rightIcon, id, ...props }, ref) => {
     const Container = leftIcon || rightIcon ? InputGroup : Box;
     return (
       <Container position="relative">
         {leftIcon && <InputLeftElement>{leftIcon}</InputLeftElement>}
         <ChakraInput
           pl={leftIcon ? 7 : undefined}
           pr={rightIcon ? 7 : undefined}
+          id={id}
           {...props}
           ref={ref}
           placeholder=" " // This is needed to make the label work as expected
         />
-        <FormLabel>{label}</FormLabel>
+        <FormLabel htmlFor={id} pointerEvents="none">
```

Comment:
Man kan argumentere for at pointerEvents="none" på en label er feil, men i dette tilfelle vil resultatet bli det samme om man klikker på labelen eller ikke. Hele labelen vil alltid være på innsiden av input-feltet, så da ser jeg ikke noe problem med å gjøre det slik.

Alternativt så kunne man brukt de interne hookene `useFormControlProps` og `useFormControl` for å hente ut riktige IDer for å binde dem sammen, men det har sin egen risikoprofil (de er ikke dekket av semver f.eks). Så summa summarum, et reflektert trade-off.